### PR TITLE
[sharing] (Run-from buttons) Adding section about run-from buttons

### PR DESCRIPTION
**Pull request by AlexJerabek**
Preview link: https://review.docs.microsoft.com/en-us/office/dev/scripts/overview/excel?branch=AlexJ-RunFromButton#create-script-buttons

### CODE REVIEW

**Comment by AlexJerabek** [comment]
@nancy-wang, could you please verify this for accuracy?

**Review by alison-mk — APPROVED** [review]
Approving, with one general comment: "the **...** menu" feels inaccessible to me. It's visually ambiguous in text and doesn't match the design which places the dots in the center. Is there another way we could type this, perhaps **···**? Can discuss offline!

**Comment by AlexJerabek** [comment]
@alison-mk  Great point about the ... menu. Let me take another stab at that (and maybe fix this build error).

**Review by alison-mk — APPROVED** [review]

File: docs/overview/excel.md [hunk 1]
```diff
@@ -68,7 +68,15 @@ More details about sharing and unsharing scripts can be in the article [Sharing
 > [!NOTE]
 > Learn more about how scripts are stored in your OneDrive in [Office Scripts file storage and ownership](script-storage.md).
 
-## Connecting Office Scripts to Power Automate
+### Create script buttons (preview)
+
+[!INCLUDE [Public preview](../includes/preview-note.md)]
+
+Help your colleagues find and run your scripts by adding script buttons to a workbook. With any script, go to the **More options (…)** menu and select **Add button**. This creates a button in the workbook that runs the associated script when pressed. It also shares the script with the workbook, so everyone with access can use your helpful automation.
```
Comment (nancy-wang) [inline docs/overview/excel.md:75]: Users can add buttons through two ways (1) the More options menu in the Script Details pane and (2) the More options menu in the Code Editor. Suggest including that info somewhere in here e.g. "With any script, go to the "More options" menu in either your Code Editor or Script Details pane and select "Add button".

File: docs/overview/excel.md [hunk 2]
```diff
@@ -68,7 +68,15 @@ More details about sharing and unsharing scripts can be in the article [Sharing
...
+
```
Comment (nancy-wang) [inline docs/overview/excel.md:76]: It's true that the 'add button' option is in the code editor's menu, but I think the screenshot shows the Script Detail pane's menu

File: docs/overview/excel.md (hunk 1 above)
Comment (nancy-wang) [reply docs/overview/excel.md:75]: Nit: should we add that it shares the script with everyone with write/edit access? a user can't run a button in read-only mode - this behavior is consistent with running scripts though

**Review by nancy-wang — COMMENTED** [review]
Added some details on where the "Add button" option is located

**Comment by alison-mk** [comment]
@nancy-wang I've addressed your feedback, can you take another look? I used "Details page" instead of "Details pane" to align with the terminology in our support docs: https://support.microsoft.com/en-us/office/sharing-office-scripts-in-excel-for-the-web-226eddbc-3a44-4540-acfe-fccda3d1122b 

Thanks!

File: docs/overview/excel.md [hunk 3]
```diff
@@ -68,7 +68,19 @@ More details about sharing and unsharing scripts can be in the article [Sharing
 > [!NOTE]
 > Learn more about how scripts are stored in your OneDrive in [Office Scripts file storage and ownership](script-storage.md).
 
-## Connecting Office Scripts to Power Automate
+### Create script buttons (preview)
+
+[!INCLUDE [Public preview](../includes/preview-note.md)]
+
+Help your colleagues find and run your scripts by adding script buttons to a workbook. With any script, go to the **More options (…)** menu in the script's Details page and select **Add button**. This creates a button in the workbook that runs the associated script when selected. It also shares the script with the workbook, so everyone with write permissions to the workbook can use your helpful automation.
```
Comment (nancy-wang) [inline docs/overview/excel.md:75]: Minor detail - they could go to the More options menu in either the details or the code editor page of the script.

Should we mention something about how to stop sharing a button or should that be part of a separate set of docs? (To stop sharing, users could either delete the button or select "Stop sharing" in the More options menu.)

**Review by nancy-wang — COMMENTED** [review]
Left a comment about how users could stop sharing the button and also where they could find the Add button option in the code editor - beyond that, looks good. thank you SO MUCH for all your work on this!!

File: docs/overview/excel.md (hunk 3 above)
Comment (AlexJerabek) [reply docs/overview/excel.md:75]: I'll add those points in. Thanks for raising them.

**Review by nancy-wang — APPROVED** [review]

**Pull request merged**